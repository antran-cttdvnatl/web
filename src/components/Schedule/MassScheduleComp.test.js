import React from 'react';
import renderer from 'react-test-renderer';
import MassScheduleComp from './MassScheduleComp';


it('MassSchedule should render correctly', () => {
    const component = renderer.create(
        <MassScheduleComp />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})