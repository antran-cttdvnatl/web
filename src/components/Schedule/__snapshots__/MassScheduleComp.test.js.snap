// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MassSchedule should render correctly 1`] = `
<section
  className="mass_schedule"
>
  <div
    className="jumbotron jumbotron-fluid"
  >
    <h1>
      Our Mass Schedule
    </h1>
    <div
      className="row"
    >
      <dl>
        <dt>
          Monday-Saturday
        </dt>
        <dd>
          7:30 AM
        </dd>
        <dd>
          9:30 AM
        </dd>
        <dd>
          11:30 AM
        </dd>
      </dl>
      <dl>
        <dt>
          Thursday-Friday
        </dt>
        <dd>
          7:00 PM
        </dd>
      </dl>
      <dl>
        <dt>
          Sunday 
        </dt>
        <dd>
          7:30 AM
        </dd>
        <dd>
          2 x 9:30 AM
        </dd>
        <dd>
          11:30 AM
        </dd>
      </dl>
    </div>
  </div>
</section>
`;
